packages/components/src/components/form/component.tsx

```diff
@@ -72,6 +79,8 @@ export class KolForm implements FormAPI {
 		);
 	}
 
+	errorListElement!: HTMLElement;
```

Comment:
Das müsste doch nullable sein, da es `undefined` bleibt, wenn es keine Fehler in der Liste gibt. Also `errorListElement?: HTMLElement;`

Weiter unten in Zeile 134 prüfst du es ja sogar.